packages/synapse-react-client/src/components/TableColumnSchemaEditor/TableColumnSchemaEditor.tsx

```diff
@@ -42,15 +49,26 @@ function _TableColumnSchemaEditor(props: TableColumnSchemaEditorProps) {
   const entity = bundle?.entity
   const originalColumnModels = bundle?.tableBundle?.columnModels
 
-  const { mutate, isLoading: isMutating, error } = useUpdateTableColumns()
+  const {
+    mutate,
+    isLoading: isMutating,
+    error,
+  } = useUpdateTableColumns({
```

Comment:
Consider trying to consolidate omitting undefined properties in some shared method called by both this component and the CreateTableViewWizard

Reply:
Could do this in `SynapseClient.createColumnModels` but hesitant to put business logic in SynapseClient